internal/repositories/dcn.go

```diff
@@ -59,3 +63,141 @@ func (r *Repository) GetDCNByName(ctx context.Context, name string) (*gmodel.Dcn
 
 	return DCNToAPI(dcn), nil
 }
+
+func (r *Repository) GetDCNs(ctx context.Context, first *int, after *string, last *int, before *string, filterBy *gmodel.DCNFilter) (*gmodel.DCNConnection, error) {
+	var limit int
+
+	if first != nil {
```

Comment:
Doyin has a helper in `helpers` that I think we can use.

Reply:
https://github.com/DIMO-Network/identity-api/blob/c4e1456c6969e4f2a4cc89ec90c8465702d7ee69/internal/repositories/vehicle_privileges.go#L119